topics/swift/index.md

```diff
@@ -1,13 +1,14 @@
 ---
-aliases: swift-3, swift3, swift4, swift-4, swift-language
-created_by: Apple Inc.
+topic: swift
+aliases: swift3, swift-3, swift4, swift-4, swift-language
+related: objective-c
 display_name: Swift
-github_url: https://github.com/apple/swift
+created_by: Apple
+logo: swift.png
 released: June 2, 2014
-short_description: Swift is a dynamic programming language that allows C, C++, and
-  Objective-C to run within the same program.
-topic: swift
+short_description: Swift is a modern programming language focused on safety, performance and expressivity.
```

Comment:
Would you add a `,` after "performance"? See https://github.com/github/explore/blob/master/docs/styleguide.md#commas.